src/index.js

```diff
@@ -1,128 +1,128 @@
-'use strict';
+'use strict'
 
-var React = require('react');
-var matchMedia = require('matchmedia');
-var hyphenate = require('hyphenate-style-name');
-var mediaQuery = require('./mediaQuery');
-var toQuery = require('./toQuery');
-var assign = require('object-assign');
+let React = require('react')
```

Comment:
Out of curiosity. Why not using `const` for variables we are sure that won't be redeclared?